docs-markdown/markdownlint-custom-rules/image.js

```diff
@@ -70,6 +71,18 @@ module.exports = {
                         }
                     }
 
+                    //lightbox source check
+                    if (!notImageComplexEndTagMatch) {
+                        const sourceMatch = "common.imageLightboxMatch.exec(content);
```

Comment:
Hi @adunndevster , missing quote resulting in the error below:
{
	"resource": "/c:/GitHub/vscode-docs-authoring/docs-markdown/markdownlint-custom-rules/image.js",
	"message": "Unterminated string literal.",
	"startLineNumber": 76,
}

Reply:
Fixed!